packages/idempotency/README.md

```diff
@@ -5,7 +5,6 @@ Powertools for AWS Lambda (TypeScript) is a developer toolkit to implement Serve
 You can use the package in both TypeScript and JavaScript code bases.
 
 - [Intro](#intro)

```

Comment:
I think we can remove the table of contents entirely and slim down the doc like we are doing in other packages, same for the `<!-- omit in toc -->` in the title